internal/linker/link_node_modules.ts

```diff
@@ -485,11 +512,28 @@ export async function main(args: string[], runfiles: Runfiles) {
   log_verbose('manifest file', modulesManifest);
   log_verbose('manifest contents', JSON.stringify({workspace, bin, root, modules}, null, 2));
 
-  // NB: resolveRoot will change the cwd when under runfiles to the runfiles root
-  const rootDir = await resolveRoot(root, runfiles);
+  // Bazel starts actions with pwd=execroot/my_wksp when under execroot or pwd=runfiles/my_wksp
+  // when under runfiles.
+  // Normalize the slashes in startCwd for easier matching and manipulation.
+  const startCwd = process.cwd().replace(/\\/g, '/');
+  log_verbose('startCwd', startCwd);
+
+  // We can derive if the process is being run in the execroot if there is a bazel-out folder.
```

Comment:
this is a different mechanism than in your other PR where you check for `execroot` in the path. Is one better than the other?

Reply:
Yes. Looking for `execroot` basename will not work in RBE. I'll fix up the other PR. Thanks!

Reply:
Actually, the other PR needs to change once it is rebased on top of this one after it lands since the location of the linker node_modules has moved here.

Reply:
https://github.com/bazelbuild/rules_nodejs/pull/1800 for future reference